### PR TITLE
[fix-#11788]add return message

### LINKED ISSUE #11788: [Bug] [ApiServer] Workflow definition page batch deletion of "online" status workflow, prompt suggestions for optimization
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

image
image

### What you expected to happen

Optimization tips

### How to reproduce

1. Online workflow; 
2. Batch deletion workflow

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuchanghai**
## Purpose of the pull request

add return message

## Brief change log

add return message
## Verify this pull request

This pull request is code cleanup without any test coverage.

this close #11788

### CODE REVIEW

**Comment by fuchanghai** [comment]
@SbloodyS  Although I have added a judgment on the online status, I don't think the above issue can be fixed, because it is necessary to judge whether it exists before it can judge whether the status is online. My initial suspicion is that he deletes the process definition after someone else has already deleted it

**Review by SbloodyS — DISMISSED** [review]
LGTM

**Comment by fuchanghai** [comment]
@SbloodyS  i find some error,we should throw new ServiceException when the workflow's states is online. because of Transactional

**Comment by fuchanghai** [comment]
@SbloodyS @caishunfeng Help reivew when you are free

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java [hunk 1]
```diff
@@ -856,7 +856,6 @@ public Map<String, Object> deleteProcessDefinitionByCode(User loginUser, long pr
             putMsg(result, Status.USER_NO_OPERATION_PERM);
             return result;
         }
-
         processDefinitionUsedInOtherTaskValid(processDefinition);
```
Comment (caishunfeng) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java:859]: It has already check the online state here.

**Review by caishunfeng — CHANGES_REQUESTED** [review]

**Comment by fuchanghai** [comment]
@caishunfeng  may be this is not a bug #11788

**Pull request closed without merging**